final-project/scripts/main.js

```diff
@@ -1,10 +1,168 @@
 import "../styles/main.scss";
+import "../styles/hero-section.scss";
+import "../styles/customer-section.scss";
+import "../styles/planManage-section.scss";
+import "../styles/features-section.scss";
+import "../styles/stats-section.scss";
+import "../styles/blog-section.scss";
+import "../styles/quotes-section.scss";
+import "../styles/faq-section.scss";
+import "../styles/clouds-section.scss";
+import "../styles/newsletter-section.scss";
+import "../styles/footer.scss";
+import "../styles/responsive.scss";
+
 
 // testing that `const` and arrow functions transpile correctly
 const before = "Hello -world!";
 const after = before
   .split("")
   .filter(char => char !== "-")
   .join("");
-console.log({ before });
-console.log({ after });
+
+var currentBlog = 0;
+var blogData;
+var customerInDisplay = 1;
+
+readBlogData();
+
+function readBlogData() {
+  let url = 'https://wt-4662f45b9eefda7172b747b28d23efdb-0.sandbox.auth0-extend.com/blog';
+  fetch(url)
+  .then(res => res.json())
+  .then((out) => {
+    blogData = out;
+    modifyBlogData(currentBlog);
+    updateSliderColor(currentBlog);
+    addButtonListenersBlog();
+    addButtonListenersCustomer();
+    mediaQueries();
+  })
+  .catch(err => { throw err });
+}
+
+function modifyBlogData(currentBlog) {
+  var textSection = document.getElementsByClassName('blog-section__content-text');
+  var titleText = textSection[0].getElementsByTagName('h1');
```

Comment:
Remember that selecting HTML nodes directly is not a good practice, using classes is preferred in case the site structure changes.